playwright-browsers.config.ts

```diff
@@ -31,10 +30,10 @@ const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
-  /* Opt out of parallel tests on CI. */
-  workers: os.cpus().length,
+  /* All CPUs on CI / half of availible CPUs when testing locally. */
+  workers: process.env.CI ? os.cpus().length : os.cpus().length / 2,
```

Comment:
I think this syntax should work and it is more readable
```suggestion
  workers: process.env.CI ? '100%' : '50%',
```

Reply:
Are these valid values?

Reply:
Looks like it from the docs: https://playwright.dev/docs/test-configuration#testing-options